RadixWallet/Clients/SubmitTransactionClient/SubmitTransactionClient+Interface.swift

```diff
@@ -10,23 +10,23 @@ public struct SubmitTransactionClient: Sendable {
 }
 
 extension SubmitTransactionClient {
-	public typealias SubmitTransaction = @Sendable (SubmitTXRequest) async throws -> TXID
-	public typealias HasTXBeenCommittedSuccessfully = @Sendable (TXID) async throws -> Void
+	public typealias SubmitTransaction = @Sendable (SubmitTXRequest) async throws -> IntentHash
```

Comment:
TXID has always been just s typealias for IntentHash, we ought to start using the correct term..

Reply:
They are semantically different things. `IntentHash` *has* `txID`, but *is* not per se. the TXID is the bech32 encoding of the hash contained in `IntentHash`. The `IntentHash` also has a `networkID`. 

We should use `txID` - refering to `intentHash.formatted(.raw)` or `intentHash. bech32EncodedTxId` (they are the same) only when we need the bech32 encoded string - and we ought to do it "as late as possible". Since it is good to retain as much info as possible (i.e. `networkID` of an `IntentHash`).

Reply:
but still, it is wrong to have a property `txID: IntentHash`, right?

Reply:
I dont care too much about a few properties with inaccurate name in grand scheme of things. We can rename them as we go if you like.

Reply:
you sounded somewhat categorical about using the correct term with your initial comment, so I thought you wanted to pursue it to the fullest.

Reply:
That was not my intent (pun intended (double pun intended)) - I forgot to write some emoji.

I just think it is good to avoid confusion - at least on type level, so "TXID" ought to be not used.